Sources/Markdown/Base/MarkupData.swift

```diff
@@ -28,6 +28,12 @@ struct MarkupIdentifier: Equatable {
     func nextSibling(from raw: RawMarkup) -> MarkupIdentifier {
         return .init(rootId: rootId, childId: childId + raw.subtreeCount)
     }
+    
+    /// Returns the identifier for the next sibling of the given raw elements.
```

Comment:
Not a fan of this being API using `[RawMarkup]`. Please go ahead and inline these as needed and remove these function declarations.

Reply:
Addressed here: 4c3fcaadab1adf02ae54638145ebae81098eae1d.

Reply:
Beautiful thank you.